quinn-udp/src/unix.rs

```diff
@@ -43,6 +43,11 @@ pub struct UdpSocketState {
 }
 
 impl UdpSocketState {
+    /// Tokio-compatible UDP socket with some useful specializations
```

Comment:
nit: this crate is completely decoupled from async runtime concerns. Maybe just say "QUIC-friendly" as in Windows below?